src/js/utils/dom.js

```diff
@@ -253,7 +258,7 @@ export function openLink(link, target, additionalOptions = {}) {
     let a = document.createElement('a');
     a.href = link;
```

Comment:
This is where the url is assigned to the element. This can trigger browser pre-fetch. `additionalOptions` would be what exactly (they could overwrite `href`, but that's not likely why we have this option)?

Reply:
Nice catch. Updating now.